app/scripts/background.js

```diff
@@ -965,17 +954,17 @@ const addAppInstalledEvent = () => {
   setTimeout(() => {
     // If the controller is not set yet, we wait and try to add the "App Installed" event again.
     addAppInstalledEvent();
-  }, 1000);
+  }, 500);
```

Comment:
Just a question: can we guarantee that the controller will be set after 0.5s here?

Reply:
I can't guarantee it, but it is recursive, so it will eventually be set. I decreased to 500 here because for one of our tests it doesn't wait long enough for the recursion to kick in before failing the test.

Reply:
Gotcha, then let's stick to this one for now 🙏🏻